### PR TITLE
chore: override configs during Init and bump core

### PR DESCRIPTION
**Pull request by evan-forbes**
## Description

bump core and override configs in the init command.

we should probably move this to the app eventually, as that is more explicit and will require fewer changes to the sdk #151

Closes: #XXXX

### CODE REVIEW

**Comment by evan-forbes** [comment]
I think failing tests might be related to github packages being degraded currently 
https://www.githubstatus.com/

File: x/genutil/client/cli/init.go [hunk 1]
```diff
@@ -79,6 +80,14 @@ func InitCmd(mbm module.BasicManager, defaultNodeHome string) *cobra.Command {
 
 			serverCtx := server.GetServerContextFromCmd(cmd)
 			config := serverCtx.Config
+
+			// default mamaki configs TODO: move to the app to be more explicit
+			// in what we are setting as defaults
+			config.Consensus.TimeoutCommit = time.Second * 15
```
[the remark this answers is not in the document]
Comment (mindstyle85) [reply x/genutil/client/cli/init.go:86]: 25s for validators

[the remark this answers is not in the document]
Comment (liamsi) [reply x/genutil/client/cli/init.go:86]: yes, 25s

[the remark this answers is not in the document]
Comment (liamsi) [reply x/genutil/client/cli/init.go:86]: 25s should yield about 30s block time imo
for devnet we had 15s landed roughly on 20s block times
so I guess 25s will yield about 30-32s block times

[the remark this answers is not in the document]
Comment (evan-forbes) [reply x/genutil/client/cli/init.go]: :+1: updated

https://github.com/celestiaorg/cosmos-sdk/pull/150/commits/4a28c405356518642026a062adb7c6279c18288f

**Review by SweeXordious — APPROVED** [review]

**Pull request merged**